controllers/commonservice_controller.go

```diff
@@ -407,12 +408,48 @@ func (r *CommonServiceReconciler) mappingToCsRequest() handler.MapFunc {
 		cmName := object.GetName()
 		cmNs := object.GetNamespace()
 		if cmName == constant.CsMapConfigMap && cmNs == "kube-public" {
-			CsInstance = append(CsInstance, reconcile.Request{NamespacedName: types.NamespacedName{Name: "common-service", Namespace: r.Bootstrap.CSData.OperatorNs}})
+			CsInstance = append(CsInstance, reconcile.Request{NamespacedName: types.NamespacedName{Name: constant.MasterCR, Namespace: r.Bootstrap.CSData.OperatorNs}})
 		}
 		return CsInstance
 	}
 }
 
+func (r *CommonServiceReconciler) mappingToCsRequestForOperandRegistry() handler.MapFunc {
+	return func(object client.Object) []reconcile.Request {
+		operandRegistry, ok := object.(*odlm.OperandRegistry)
+		if !ok {
+			// It's not an OperandRegistry, ignore
+			return nil
+		}
+
+		if shouldReconcile(operandRegistry) {
+			// Enqueue a reconciliation request for the corresponding CommonService
+			return []reconcile.Request{
+				{NamespacedName: types.NamespacedName{Name: operandRegistry.Name, Namespace: operandRegistry.Namespace}},
```

Comment:
I am wondering whether we should queue the request for CommonService CR under operator namespace?

Usually the `operandRegistry.Namespace` is service namespace.